packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/index.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Overview of the mobile Identity Engine SDK
+---
+
+<div class="oie-embedded-sdk">
+
+Design the implementation of the sign-in flow for your mobile app by understanding the representation of the flow, and with examples of the common parts.
+
+<ApiLifecycle access="ie" /><br>
+
+## Introduction
+
+Integrate the Okta sign-in flow into your app in one of two ways:
+
+- Redirect the request to Okta which handles the flow in its own window (the redirect model).
+- Handle each step in the flow by building the appropriate user interface elements, capturing the information, and updating Okta (the embedded model).
+
+This guide is an overview of building your own user interface using the embedded model. For information on the redirect model, see <StackSnippet snippet="redirectquickstart" inline />.
+
+## Sign-in flow
+
+Okta supports many ways of authenticating the identity of a user during the sign-in flow. An Okta org administrator creates _policies_, different mixes of authentication methods (also called factors), and assigns them to apps, people, groups, and more. Policies also configure how many authentication methods a sign-in flow requires (multifactor), and which methods are optional. This results in a large number of possible combinations for the different factors and steps to complete the sign-in flow.
```

Comment:
I want to check in on something because there's a reason I structured the paragraph as I did and used the terms I did. This article is for someone who may know nothing about the terminology of the domain (authenticators, factors, etc.) The paragraph services two functions: it introduces them to the terminology, and it builds to the climax of "implement a dynamic UI."

It looks like you're recommending using the term "authenticators". If that's the case, then how about this:

> Okta supports many ways of authenticating the identity of a user during the sign-in flow. An Okta org administrator creates _policies_, different mixes of authenticators, or ways of authenticating a user (also called factors), and assigns them to apps, people, groups, and more. Policies also configure how many authenticators a sign-in flow requires (multifactor), and which are optional. This results in a many possible combinations of the different authenticators, the steps in the sign-in flow.

Reply:
I actually like my revision better since it doesn't use factors and authenticators in unclear ways. However, for sake of time - I'll let you make the call. We can always go back and revise it if needed.

Reply:
We're getting there. It's important to remember the goal of the paragraph and it's role in the overall story, the flow of this guide. This paragraph:

- Defines the key Okta org concepts associated with the sign-on flow: policies, authenticators.
- I also used it to define methods, factors, multifactor because that's my understanding of what these terms are... it sounds like I have an incorrect association, and if so, please help me understand them so that the terms can be introduced appropriately. It's important that a developer gets a rudimentary understanding of factors and multifactor before the next paragraph (and it's OK to introduce an interim paragraph if that makes sense.)
- Worries the reader by making it clear there are lots of different authenticators that can occur in many different ways during the flow, that may or may not be required, and that can be changed at any time by someone else.

For example, that's why I use the word "many" in the first sentence. It's the first point for forming an association between ways of authenticating, and that there are many (it takes at least 3 exposures to an association expressed somewhat differently to form the association in short-term memory... The strength of the association depends on the engagement level of the person.)